### ISSUE TITLE
Startup error in nvm_lsp

### ISSUE THREAD

**Issue by stonemaster** [rafi/vim-config#132]
I have the following startup error with current master (975a2e0):

```Error detected while processing /home/andre/.cache/vim/dein/repos/github.com/neovim/nvim-lsp/plugin/nvim_lsp.vim:                                                                                                                                                                
line   11:                                                                                                                                                                                                                                                                       
E5105: Error while calling lua chunk: ...n/repos/github.com/neovim/nvim-lsp/lua/nvim_lsp/util.lua:300: 
 attempt to index local 'fn' (a nil value)
```

I am on Ubuntu 19.10 with nvim 0.4.3.

The error is display each time I startup nvim.

**Comment by rafi** [comment]
I guess LSP functions are available only from 0.5.0, I've pushed fef37b0 to verify that before installing. Let me know if that doesn't help...

**Comment by stonemaster** [comment]
Thanks! That helped.
